integrations/Braze/browser.js

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable consistent-return */
```

Comment:
Add the relevant test cases for the new dedupe logic

Reply:
Yeah, will update.

Reply:
updated